js/Localization.js

```diff
@@ -0,0 +1,114 @@
+import i18n from 'i18n-js';
+import * as RNLocalize from 'react-native-localize';
+
+i18n.translations = {
+  en: {
+    add_first_site: '+ Add your first site',
+    add_site: 'Add Site',
+    add_sites: 'Add Discourse sites to keep track of.',
+    add: '+ Add',
+    all: 'All',
+    and: 'and',
+    approved_commits: '%{count} commits approved',
```

Comment:
Hmm, looks like you are using `i18n-js` behind the scenes. Our implementation in core is based on an older version of it, so I'm assuming it still supports pluralization. If that's the case, it might be a good idea, to split the string into two:

```suggestion
    approved_commits: {
      one: '%{count} commit approved',
      other: '%{count} commits approved',
    },
```

This applies to the other strings with `%{count}` as well.